scripts/cert-gen

```diff
@@ -33,7 +33,7 @@ echo $password > $passfile
 
 # Generate the CA
 openssl genrsa -aes256 -passout file:$passfile -out ca-key.pem 2048
-openssl req -new -x509 -passin file:$passfile -days 365 -key ca-key.pem -sha256 -out ca.pem -subj "/C=/ST=/L=/O=/OU=/CN=example.com"
+openssl req -new -x509 -passin file:$passfile -days 365 -key ca-key.pem -sha256 -out ca.pem -subj "/C=US/ST=NY/L=New York/O=Example/OU=X/CN=example.com"
```

Comment:
right, this was one of my original complaints with the fedora iso, as the same issue happened also with the centos iso. it enforces the information to be 'realistic'.

However, can we at least use `minishift.io` instead. `example.com` is an owned/existing domain. Also, the locality can be something like `Interwebs` that feels less realistic, but acceptable.

Reply:
It also complained about the IP (127.0.0.1)